### PR TITLE
style and es6 changes

### PR DESCRIPTION
**Pull request by jray**
Proposed changes for styles and using current [node LTS](https://nodejs.org/en/blog/release/v4.2.0/) version which comes with a few es6 features.

### CODE REVIEW

File: {{ cookiecutter.repo_name }}/gulpfile.js [hunk 1]
```diff
@@ -87,70 +95,55 @@ gulp.task('start', ['nunjucks', 'sass', 'extras', 'watchify'], function() {
...
-
 gulp.task('default', ['browserify', 'nunjucks', 'sass', 'extras']);
-gulp.task('prod', ['banner', 'rev:replace', 'minify', 'critical']);
 
-gulp.task('build-dev', function(done) {
-  runSequence('clean',
-              ['default', 'start'],
-              done);
+gulp.task('build-dev', (done) => {
+  const sequence = ['default', 'start'];
```
Comment (akrawchyk) [inline {{ cookiecutter.repo_name }}/gulpfile.js]: Do we need to make a variable here? Would rather just put the array inline in the `runSequence` call I think.

File: {{ cookiecutter.repo_name }}/gulpfile.js [hunk 2]
```diff
@@ -87,70 +95,55 @@ gulp.task('start', ['nunjucks', 'sass', 'extras', 'watchify'], function() {
...
 });
 
-gulp.task('clean', function() {
+gulp.task('clean', () => {
   return del('./public/');
 });
 
-
 gulp.task('default', ['browserify', 'nunjucks', 'sass', 'extras']);
-gulp.task('prod', ['banner', 'rev:replace', 'minify', 'critical']);
```
Comment (akrawchyk) [inline {{ cookiecutter.repo_name }}/gulpfile.js]: Can you add this line back?

File: {{ cookiecutter.repo_name }}/gulpfile.js [hunk 3]
```diff
@@ -87,70 +95,55 @@ gulp.task('start', ['nunjucks', 'sass', 'extras', 'watchify'], function() {
...
-gulp.task('banner', ['browserify'], function() {
+gulp.task('banner', ['browserify'], () => {
   return gulp.src(['banner.txt', './public/js/bundle.js'])
-  .pipe(concat('bundle.js'))
-  .pipe(gulp.dest('./public/js/'));
+    .pipe(concat('bundle.js'))
+    .pipe(gulp.dest('./public/js/'));
 });
 
-gulp.task('minify', ['rev:replace', 'critical'], function() {
```
Comment (akrawchyk) [inline {{ cookiecutter.repo_name }}/gulpfile.js]: Can you add the dependency on the `critical` task back?

**Comment by akrawchyk** [comment]
Looks good, left a few comments but I think most of them would be resolved with a rebase.

File: {{ cookiecutter.repo_name }}/src/js/{{ cookiecutter.repo_name }}.js [hunk 4]
```diff
@@ -1,3 +1,6 @@
+
```
Comment (akrawchyk) [inline {{ cookiecutter.repo_name }}/src/js/{{ cookiecutter.repo_name }}.js]: Why intro whitespace?

File: {{ cookiecutter.repo_name }}/gulpfile.js (hunk 3 above)
Comment (jray) [inline {{ cookiecutter.repo_name }}/gulpfile.js]: Yea I think what happened was I brought all changes from the franklin-dashboard gulpfile - I assumed they were the same with no mods. I'll get this back in.

File: {{ cookiecutter.repo_name }}/gulpfile.js (hunk 2 above)
Comment (jray) [inline {{ cookiecutter.repo_name }}/gulpfile.js]: Same as above the the dashboard changes

File: {{ cookiecutter.repo_name }}/gulpfile.js (hunk 1 above)
Comment (jray) [inline {{ cookiecutter.repo_name }}/gulpfile.js]: Read a bit better to me. But it's certainly not required.

File: {{ cookiecutter.repo_name }}/src/js/{{ cookiecutter.repo_name }}.js (hunk 4 above)
Comment (jray) [inline {{ cookiecutter.repo_name }}/src/js/{{ cookiecutter.repo_name }}.js]: Must have been one of my formatters

**Pull request merged**